src/schedule/zephyr_ll.c

```diff
@@ -14,6 +14,10 @@
 #include <rtos/task.h>
 #include <sof/lib/perf_cnt.h>
 #include <zephyr/kernel.h>
+#include <ipc4/base_fw.h>
+#ifdef CONFIG_SOF_TELEMETRY
+#include <sof/debug/telemetry/telemetry.h>
+#endif
```

Comment:
`telemetry.h` does not seem to have anything out of the ordinary so I see no reason to make this conditional. Kconfig combinatorial explosion and lack of coverage is already painful enough: no need to add even more coverage gaps and realize even later than some symbols haven't been namespaced and collide.